File: foodit/src/app/food.ts
import { Url } from 'url';
import { Route } from '@angular/compiler/src/core';
import { Source } from 'webpack-sources';
import { UrlMatcher } from '@angular/router';

export class Food {
    id: number;
    seller: string;
    name: string;
    price:number;   
    quantity:number;
    description: string;
    placeToMeet: string;
    image: string;
}

File: food.api/Models/Food.cs
using System;

namespace Foodit.Api.Models{


 public class Food {
    public int Id {get;set;}

    public string Name{get;set;}
    public string Description {get;set;}
    public int Price{get;set;}

    public int Quantity{get;set;}

    public string image{get;set;}


   //  public string MyTransId  {get;set;}
   //      public int Id { get; internal set; }

       
   //      public Transaction MyTrans {get;set;}
    }
}